compiler/rustc_errors/src/emitter.rs

```diff
@@ -224,23 +224,24 @@ pub trait Emitter: Translate {
 
     fn source_map(&self) -> Option<&Lrc<SourceMap>>;
 
+    /// njn: update comment
```

Comment:
What did you have in mind for this?

Reply:
Whoops! I updated the comment but forgot to remove the comment. I will remove it and update.